samples/powerstore_v260_k8s_124.yaml

```diff
@@ -56,27 +63,20 @@ spec:
         - name: X_CSI_NFS_ACLS
           value: "0777"
 
-
       # nodeSelector: Define node selection constraints for controller pods.
       # For the pod to be eligible to run on a node, the node must have each
       # of the indicated key-value pairs as labels.
       # Leave as blank to consider all nodes
       # Allowed values: map of key-value pairs
       # Default value: None
       nodeSelector:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-      #  node-role.kubernetes.io/master: ""
       # Uncomment if nodes you wish to use have the node-role.kubernetes.io/control-plane taint
       #  node-role.kubernetes.io/control-plane: ""
 
       # tolerations: Define tolerations for the controllers, if required.
       # Leave as blank to install controller on worker nodes
       # Default value: None
       tolerations:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-      # - key: "node-role.kubernetes.io/master"
```

Comment:
Just In case if we customer install this CSI-operator version and wants to install CSI-powerstore v2.4, Are we good to not have this one?
Just wanted to make sure that none of the min k8 supported version needs this taint labels.
Note: If you are using a version of Kubernetes after 1.18, this taint will not be available.

Reply:
Updated.
Thank You.!